Formula/wash.rb

```diff
@@ -1,7 +1,7 @@
 class Wash < Formula
   desc "WAsmcloud SHell - A multi-tool for various wasmCloud activities as well as an interactive REPL for a wasmCloud runtime host."
   homepage "https://wasmcloud.dev"
-  url "https://github.com/wasmCloud/wash/archive/v0.4.2.tar.gz"
+  url "https://github.com/wasmCloud/wash/archive/v0.4.3.tar.gz"
   sha256 "c4c1e4f6b6821903c51f3a23068593125d470bebfd5dd779ac3071a37ec3a1a3"
```

Comment:
This sha is for the tar.gz, so it needs to be bumped as well....
`c703e76e2da99ee6fd1ea2f33736d78c495e4d86d68d0c2669dd3393e4c0c200`

Reply:
@jordan-rash how did you compute that sha? for some reason on mac `shasum -a 256 blah.tar.gz` gives a different checksum 🤔

Reply:
Yea, I think I sha'd the wrong file 😜
```bash
❯ shasum -a256 v0.4.3.tar.gz
a80092b88c1d27153fc0c314f186dc1973809c7ec3adcc9b352f41285a864e9d  v0.4.3.tar.gz
```

Reply:
Oh I released a newer version (because of a bug) so you probably did it correctly.. but I tried to sha the `0.4.2` version and got: 
```
098b61b8367e36c633db4ff09edc2b27ca2d9056690453ba4dfd1891b67a59da
```

EDIT: I'm dumb, I tried to `curl` the tar.gz and I got a 404 page downloaded instead, so I got the hash of a 404. Carry on :)

Reply:
Ok updated to the correct hash @jordan-rash